docs/content/docs/connectors/datastream/formats/parquet.md

```diff
@@ -0,0 +1,67 @@
+---
+title:  "Parquet"
+weight: 4
+type: docs
+aliases:
+- /dev/connectors/formats/parquet.html
+- /apis/streaming/connectors/formats/parquet.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# Parquet formats
```

Comment:
```suggestion
# Parquet format
```

Reply:
Well actually there are several `ParquetRowInputFormat`, `ParquetPojoInputFormat`, `ParquetAvroInputFormat`, hence the plural.

Reply:
Are they not different implementations to read the same Parquet format? There's only 1 Parquet format specification I would guess (also looking at https://github.com/apache/parquet-format ?)

Reply:
Yes they read the same way the parquet files (ParquetReader impl is common), they just output different object types (Avro GenericRecords, Flink Row or java Pojo). So they are considered the same format then ?